appointment/lambda_function.py

```diff
@@ -10,15 +10,15 @@
 def lambda_handler(event, context):
     
     #Getting the values to put into the row
-    student_id = int(event['student_id'])
-    supporter_id = int(event['supporter_id'])
-    appt_date = event['appt_date']
-    start_time = event['start_time']
-    duration_min = int(event['duration'])
-    type = int(event['type'])
-    cancelled = bool(event['cancelled'])
-    rating = int(event['rating'])
-    recommended = bool(event['recommended'])
+    student_id = int(json.loads(event["body"])["student_id"])
+    supporter_id = int(json.loads(event["body"])["supporter_id"])
+    appt_date = json.loads(event["body"])["appt_date"]
+    start_time = json.loads(event["body"])["student_id"]
```

Comment:
typo: "student_id" should be "start_time"

Reply:
Oh good catch

Reply:
yeah it was throwing me for a loop while testing it